src/app/models/grid.model.ts

```diff
@@ -15,12 +16,29 @@ export class Grid {
                 this.cells[r][c] = new Cell(this, r, c);
             }
         }
+        this.outOfBoundsCell = new Cell(this, -1, -1);
     }
 
-    getCell(row: number, col: number): Cell{
+    getCell(row: number, col: number): Cell {
+        return this.getCellBounded(row, col);
+    }
+
+    getCellUnbounded(row: number, col: number): Cell {
```

Comment:
I think this method is not used anywhere, so we could remove it, right?

Reply:
Yep, wanted to leave it for the sake of having a choice to switch between the two with a single function name change.